src/warnet/constants.py

```diff
@@ -98,3 +98,8 @@
     f"helm upgrade --install grafana-dashboards {CHARTS_DIR}/grafana-dashboards --namespace warnet-logging",
     f"helm upgrade --install --namespace warnet-logging loki-grafana grafana/grafana --values {MANIFESTS_DIR}/grafana_values.yaml",
 ]
+
+# Helm binary
+HELM_LATEST_URL = "https://get.helm.sh/helm-latest-version"
```

Comment:
I think we should pin a version and also commit to the expected checksums:
https://github.com/helm/helm/releases

Reply:
I decided to bless the latest version and always pull it: https://github.com/bitcoin-dev-project/warnet/pull/610/commits/07104ee988914e568655dde1e324f752ca6887f8